pkg/agent/openflow/client.go

```diff
@@ -825,12 +825,22 @@ func (c *client) InstallTraceflowFlows(dataplaneTag uint8) error {
 	for _, conj := range c.policyCache.List() {
 		for _, flow := range conj.(*policyRuleConjunction).metricFlows {
 			if flow.IsDropFlow() {
+				copyFlowBuilder := flow.CopyToBuilder(priorityNormal+2, false)
+				// Generate both IPv4 and IPv6 flows if the original drop flow doesn't match IP/IPv6
+				if flow.FlowProtocol() == "" {
```

Comment:
I see metrics drop flow doesn't have protocol set and it's not optional, then do we need to check this?

Besides, why did it fail when we didn't match protocol?

Reply:
This is a standard flow duplication process, we don't need to make sure if existing flows in cache will match IP/IPv6 or not, just checking the protocol is fine.
DSCP is a field in IP/IPv6 header, to match DSCP requires IP/IPv6 match in a flow.

Reply:
Thanks for explanation. then should the default drop flow have the same process? It adds IPv4 flow only when the drop flow doesn't have protocol set.
I think it's worth to add comments to explain why we making two traceflow flows for a single drop flow, and have an e2e test as having an antrea policy with drop action is a very common use case.

Reply:
For default drop flow , the change is in #1579 to add full support for Trace flow IPv6, but this patch may not merge in this release.

Reply:
Have the suggestions for comments and e2e tests already been addressed?

Reply:
Yes, the change is in the second commit.